daemons/fenced/pacemaker-fenced.c

```diff
@@ -1375,23 +1376,26 @@ main(int argc, char **argv)
 
         printf("  <parameter name=\"%s\" unique=\"0\">\n",
                PCMK_STONITH_DELAY_MAX);
-        printf
-            ("    <shortdesc lang=\"en\">Enable a random delay for stonith actions and specify the maximum of random delay.</shortdesc>\n");
-        printf
-            ("    <longdesc lang=\"en\">This prevents double fencing when using slow devices such as sbd.\n"
-             "Use this to enable a random delay for stonith actions.\n"
-             "The overall delay is derived from this random delay value adding a static delay so that the sum is kept below the maximum delay.</longdesc>\n");
+        printf("    <shortdesc lang=\"en\">Enable a random delay for "
+               "fencing actions and specify the maximum of random "
+               "delay.</shortdesc>\n");
```

Comment:
"specify the maximum of random delay" seems weird to me.  Maybe it's just that the word "of" should be removed.

Reply:
Will update in separate PR